dist/src/server/database/db.js

```diff
@@ -20,41 +20,12 @@ mongoose
     .catch((err) => {
     console.log(`Error connecting to Mongo DB: ${err}`);
 });
-// const connectMongoose = async () => {
-// 	try {
-// 		await mongoose.connect(
-// 			mongoURI,
-// 			{
-// 				useNewUrlParser: true,
-// 				useUnifiedTopology: true,
-// 				autoReconnect: true,
-// 				dbName: 'zeus',
-// 			},
-// 			() => console.log('🦆🦆🦆🦆🦆🦆🦆🦆  Mongoose is connected')
-// 		);
-// 	} catch (err) {
-// 		console.error('Error connecting to Mongoose:', err);
-// 	}
-// };
-// connectMongoose();
-// mongoose.connect(
-// 	mongoURI,
-// 	{
-// 		useNewUrlParser: true,
-// 		useUnifiedTopology: true,
-// 		autoReconnect: true,
-// 		dbName: 'zeus',
-// 	},
-// 	() => console.log('🦆🦆🦆🦆🦆🦆🦆🦆  Mongoose is connected')
-// );
-const Schema = mongoose.Schema;
 //user schema
 const userSchema = new mongoose.Schema({
     email: { type: String, required: true },
-    username: { type: String, required: true },
     password: { type: String, required: true },
 });
 //Collection name is Users
-const User = mongoose.model('Users', userSchema);
+const Users = mongoose.model('Users', userSchema);
 // Export db
-module.exports = User;
+module.exports = Users;
```

Comment:
Great job on the Bcrypt John! Did you mean to include the db.js file?

Reply:
Thanks! I didn't realize I sent it again. It was deleted from my branch a while ago. Is it possible to remove it?

Reply:
Just confirming there's no db.js file in my repo or branch. So I don't think merging will effect anything.